schema.graphql

```diff
@@ -235,3 +236,10 @@ enum FeeMovementType {
   Withdraw
   Subscriptions
 }
+
+type JurorTreasuryToken @entity {
+  id: ID!
+  token: ERC20!
+  juror: Juror!
+  balance: BigInt!
+}
```

Comment:
What about `TreasuryBalance`?

Reply:
Are you suggesting a name change? I don't really like it, because this entity it's just a bridge for the many to many relationship between Juror and Token entities, so I like that the name reflects that.

Reply:
> Are you suggesting a name change?

Yes, I did not understand `JurorTreasuryToken` at the beginning, I needed to read through it a few times to understand what it was doing.  

This is supposed to provide the balances stored in the treasury for each juror right? At least I would rename it to `JurorTreasuryBalance`, but not token.

Reply:
Ok, I added `Balance` to the name.